src/Activity.js

```diff
@@ -65,24 +76,25 @@ class Activity {
       return `${userName}: ${listItem[userID]}`
     })
   }
-  showcaseWinner(user, date, userRepo) {
+  showcaseWinner(user, date, userRepo) { // not tested returns winner of steps challenge of friends
     let namedList = this.showChallengeListAndWinner(user, date, userRepo);
     let winner = this.showChallengeListAndWinner(user, date, userRepo).shift();
     return winner;
   }
-  getStreak(userRepo, id, relevantData) {
-    let data = this.activityData;
+
+  getStreak(userRepo, id, dataType) { //return 3-day increasing streak for a users step count or active minutes
+    let data = this.activityData; // this is redundant
     let sortedUserArray = (userRepo.makeSortedUserArray(id, data)).reverse();
     let streaks = sortedUserArray.filter(function(element, index) {
       if (index >= 2) {
-        return (sortedUserArray[index - 2][relevantData] < sortedUserArray[index - 1][relevantData] && sortedUserArray[index - 1][relevantData] < sortedUserArray[index][relevantData])
+        return (sortedUserArray[index - 2][dataType] < sortedUserArray[index - 1][dataType] && sortedUserArray[index - 1][dataType] < sortedUserArray[index][dataType])
```

Comment:
Much better naming for the data being used.

Reply:
I agree I think that we need to make these as consistant as possible